XBVC/emitters/c_emitter.py

```diff
@@ -44,20 +63,12 @@ def gen_bitvec_encode(indent, name, container_type, is_indexed=False):
 
 
 def gen_float_bitvec_encode(indent, name, is_indexed=False):
-    result = (
-        "{indent}sf = split_float(src->{name}{index_marker}, MAX_PRECISION);\n"
-        "{indent}index += xbvc_encode_bit_vector(&sf.whole, &dest[index], E_32);\n"
-        "{idx_check}"
-        "{indent}index += xbvc_encode_bit_vector(&sf.frac, &dest[index], E_32);\n"
-        "{idx_check}"
-        .format(
-            indent=' ' * indent,
-            name=name,
-            index_marker='[i]' if is_indexed else '',
-            idx_check=gen_index_return_check(indent)
-        )
+    result = FLOAT_ENCODE_STR.format(
```

Comment:
This is a nice refactor.

Reply:
I mean, I can't draw a hard 80 char line in the sand yesterday and then not follow it :D

Reply:
I mean, I can't draw a hard 80 char line in the sand yesterday and then not follow it :D